qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1149,35 +853,6 @@ def _run(
             private=qrt_options.private,
         )
 
-    def check_pending_jobs(self) -> None:
```

Comment:
how is this functionality implemented on the cloud channels?

Reply:
It's not possible on the cloud channels, this method only worked for the open h/g/p on the old platform. Nevertheless, since it's publicly documented I'll add it back with a deprecation warning.